vm-setup/roles/v1aX_integration_test/tasks/verify.yml

```diff
@@ -48,6 +48,22 @@
                       - name: IP_AUTODETECTION_METHOD
                         value: "cidr={{ EXTERNAL_SUBNET_V4_HOST }}/{{ EXTERNAL_SUBNET_V4_PREFIX }}"
 
+  - name: Wait for all control-plane pods become created
+    shell: "kubectl get pod --namespace=kube-system --selector tier=control-plane --output=jsonpath='{.items[*].metadata.name}'"
+    register: control_plane_pods_created
+    until: item in control_plane_pods_created.stdout
+    retries: 30
+    delay: 10
+    with_items:
+      - etcd
+      - kube-apiserver
+      - kube-controller-manager
+      - kube-scheduler
+
+  - name: Wait for control-plane pods become ready
+    shell: "kubectl wait --namespace=kube-system --for=condition=Ready pods --selector tier=control-plane --timeout=600s"
+    register: control_plane_pods_ready
+
```

Comment:
Do we need to have shell commands here ? I was under the impression that these could be replace by k8s ansible modules ?

Reply:
done